source/EnhanceOption.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+namespace dungeon {
+
+enum struct EnhanceOption {
+  NORMAL_ATTACK,
+  SKILL,
+  EXIT
```

Comment:
Don't put exit here :hankey: It is not a "EnhanceOption"

Reply:
I suppose exit will be an option in the enhance dialog? Because most of the time you won't enhance for a single time so it's better to repeat until enhancement done.

Reply:
I see. The "option" is refer to the user action instead of "enhance" target.